src/GitLink/Extensions/RepositoryExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace GitLink
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Catel;
+    using Catel.IO;
+    using LibGit2Sharp;
+
+    internal static class RepositoryExtensions
+    {
+        internal static string GetRepoNormalizedPath(this Repository repository, string path)
+        {
+            Argument.IsNotNull(nameof(repository), repository);
```

Comment:
I wonder if fody is still running in my fork... I'll have to check.
IMO, something like fody's approach made sense in C# 5. But with `nameof`, I prefer the built-in support that everyone's familiar with.